qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -171,16 +169,14 @@ client.createCollection("{collection_name}", {
 });
 ```
 
-We recommend selecting the number of shards as a factor of the number of nodes you are currently running in your cluster.
+We recommend setting the number of shards to be a multiple of the number of nodes you are currently running in your cluster.
 For example, if you have 3 nodes, 6 shards could be a good option.
 
-### Cluster scaling
+Shards are evenly distributed across all existing nodes when a collection is first created, but Qdrant does not automatically rebalance shards if your cluster size changes (since this is an expensive operation on large clusters). See the next section for how to move shards after scaling operations.
```

Comment:
```suggestion
Shards are evenly distributed across all existing nodes when a collection is first created, but Qdrant does not automatically rebalance shards if your cluster size or replication factor changes (since this is an expensive operation on large clusters). See the next section for how to move shards after scaling operations.
```